docs/proposals/20201020-management-cluster-operator.md

```diff
@@ -0,0 +1,1063 @@
+---
+title: Management Cluster Operator
+authors:
+  - "@fabriziopandini"
+  - "@wfernandes"
+reviewers:
+  - "@vincepri"
+  - "@ncdc"
+  - "@justinsb"
+  - "@detiber"
+  - "@CecileRobertMichon"
+creation-date: 2020-09-14
+last-updated: 2020-10-20
+status: implementable (aspirational :wink:)
+see-also:
+https://github.com/kubernetes-sigs/cluster-api/blob/master/docs/proposals/20191016-clusterctl-redesign.md
+---
+
+# Management cluster operator
```

Comment:
I've lost access to the google doc with my previous comment but what did we conclude on the phrasing "management cluster" operator vs. "providers"?

Reply:
I seem to have lost access to that doc as well. I think @fabriziopandini might be the owner of the document.
But I remember providing some suggestions although that was a while ago 😄 . 
I'm not too attached to the name of this operator and I'm open to suggestions.

I believe the argument was since this essentially operating on the providers, it should be called like  _Provider Operator_. As per our Glossary, the definition of Management Cluster is _The cluster where one or more Infrastructure Providers run, and where resources (e.g. Machines) are stored. Typically referred to when you are provisioning multiple workload clusters._ 
From this definition, this operator is not necessarily operating on the objects stored within the management clusters but rather just the providers.

So I'm ok with calling it the **CAPI Provider Operator**. I'm open to suggestions from others 🙂  WDYT?

Reply:
I like "CAPI Provider Operator"

what do others think?

Reply:
I'm ok with CAPI Provider Operator

Reply:
Technically we also manage Cluster API itself, but we can see it as a provider? 😄

Reply:
It's consistent with the naming of the types, CAPI is a "CoreProvider"